### PR TITLE
Update6

### PR DESCRIPTION
**Pull request by mhdostal**
The update6 branch includes:

- Augmented reality component and example 
- Carthage package manager support
- SwiftLint support for both Toolkit components and the Example app
- Cleanup of the JobManager component for code consistency and style
- Minor internal tweaks

### CODE REVIEW

**Comment by mhdostal** [comment]
@njarecha @nixta All of this code has been reviewed prior to merging into the `update6` branch, so this is just a sanity check that all looks OK.

File: README.md [hunk 1]
```diff
@@ -51,7 +69,7 @@ Find a bug or want to request a new feature?  Please let us know by submitting a
 Esri welcomes contributions from anyone and everyone. Please see our [guidelines for contributing](https://github.com/esri/contributing).
 
 ## Licensing
-Copyright 2017 Esri
+Copyright 2017 - 2019 Esri
```
Comment (nixta) [inline README.md]: Do we update the Copyright range or just stick with 2017?

File: Examples/ArcGISToolkitExamples/Misc/CalibrationView.swift [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+    /// Handle an elevation slider value-changed event.
+    ///
+    /// - Parameter sender: The slider tapped on.
+    @objc
+    func elevationChanged(_ sender: UISlider) {
+        if elevationTimer == nil {
+            // Create a timer which elevates the camera when fired.
+            elevationTimer = Timer(timeInterval: 0.25, repeats: true) { [weak self] (_) in
+                let delta = self?.joystickElevation() ?? 0.0
+//                print("elevate delta = \(delta)")
```
Comment (nixta) [inline Examples/ArcGISToolkitExamples/Misc/CalibrationView.swift]: Stands out a bit. I know it's just an example view, but should we check in commented out code?

File: Examples/ArcGISToolkitExamples/Misc/CalibrationView.swift [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+    /// Handle an heading slider value-changed event.
+    ///
+    /// - Parameter sender: The slider tapped on.
+    @objc
+    func headingChanged(_ sender: UISlider) {
+        if headingTimer == nil {
+            // Create a timer which rotates the camera when fired.
+            headingTimer = Timer(timeInterval: 0.1, repeats: true) { [weak self] (_) in
+                let delta = self?.joystickHeading() ?? 0.0
+//                print("rotate delta = \(delta)")
```
Comment (nixta) [inline Examples/ArcGISToolkitExamples/Misc/CalibrationView.swift]: As above, should we check in commented out code?

File: README.md (hunk 1 above)
Comment (mhdostal) [reply README.md]: @dg0yal said it was OK to use the range.

File: Examples/ArcGISToolkitExamples/Misc/CalibrationView.swift (hunk 3 above)
Comment (mhdostal) [reply Examples/ArcGISToolkitExamples/Misc/CalibrationView.swift]: same as above.

File: Examples/ArcGISToolkitExamples/Misc/CalibrationView.swift (hunk 2 above)
Comment (nixta) [reply Examples/ArcGISToolkitExamples/Misc/CalibrationView.swift]: Yep. It's totally fine. It just stood out as I scrolled through :)

File: README.md (hunk 1 above)
Comment (nixta) [reply README.md]: Perfect.

File: Toolkit/ArcGISToolkit/AR/ArcGISARView.swift [hunk 4]
```diff
@@ -0,0 +1,529 @@
...
+        locationDataSource?.stop()
+        isTracking = false
+    }
+    
+    // MARK: Private
+    
+    /// Operations that happen after device tracking has started.
+    fileprivate func finalizeStart() {
+        DispatchQueue.main.async { [weak self] in
+            guard let strongSelf = self else { return }
```
[the remark this answers is not in the document]
Comment (mhdostal) [reply Toolkit/ArcGISToolkit/AR/ArcGISARView.swift]: Good catch, thanks.

**Pull request merged**